src/main/java/com/github/vfyjxf/nee/NEINeeConfig.java

```diff
@@ -139,11 +138,6 @@ private void installThaumicEnergisticsSupport() {
     }
 
     private void installPatternTerminalExSupport(Set<String> identifiers) {
-        try {
```

Comment:
Why did you remove this check? You're converting a soft dependency to a hard dependency by doing this

Reply:
![image](https://user-images.githubusercontent.com/57050655/190916711-0cbbb238-6c43-4788-923d-b1e8a68b36dd.png)

applied energistics is already a hard dependency

Reply:
terminalex doesn't exist in vanilla ae2 so this is for compat with non-gtnh ae2 which I guess we don't really need to keep?

Reply:
ah didn't know about that but yeah let it crash >:D